daprdocs/content/en/dotnet-sdk-docs/dotnet-workflow/dotnet-workflow-howto.md

```diff
@@ -0,0 +1,153 @@
+---
+type: docs
+title: "How to: Author and manage Dapr Workflow in the .NET SDK"
+linkTitle: "How to: Author & manage workflows"
+weight: 100000
+description: Learn how to author and manage Dapr Workflow using the .NET SDK
+---
+
+Let's create a Dapr workflow (`Workflow`) and invoke it using the console. In the provided purchase order processing workflow example, the console prompts provide directions on how to both purchase and restock items. In this guide, you will:
+
+- Create an ASP.NET application ([WorkflowConsoleApp](./WorkflowConsoleApp)). 
+- Utilize the .NET workflow SDK and API calls to start and query workflow instances.
+
+In the .NET example project:
+- The main `Program.cs` file contains the setup of the app, including the registration of the workflow and workflow activities. 
+- The workflow definition is found in the `Workflows` directory. 
+- The workflow activity definitions are found in the `Activities` directory.
+
+## Prerequisites
+
+- [.NET 6+](https://dotnet.microsoft.com/download) installed
+- [Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+- [Initialized Dapr environment](https://docs.dapr.io/getting-started/install-dapr-selfhost/)
+- [Dapr .NET SDK](https://github.com/dapr/dotnet-sdk/)
+
+## Run the application
+
+You'll need two terminal windows to run the workflow web app locally.
+
+In the first terminal window, from the `WorkflowConsoleApp` directory, run the following command to start the program itself:
+
+```sh
+dotnet run
+```
+
+Next, in a separate terminal window, start the Dapr sidecar:
+
+```sh
+dapr run --app-id wfapp --dapr-grpc-port 4001 --dapr-http-port 3500
+```
+
+Dapr listens for HTTP requests at `http://localhost:3500`.
+
+## Start a workflow
+
+To start a workflow, you have two options:
+
+1. Follow the directions from the console prompts.
+1. Use the workflow API and send a request to Dapr directly. 
+
+This guide focuses on the workflow API option. Examples are included both below and in the `WorkflowConsoleApp`>`demo.http` file.
```

Comment:
`WorkflowConsoleApp`>`demo.http` feels like a weird way to reference the file to me. Is it in a directory called `WorkflowConsoleApp`? If so, can we use `/` or `\`? Or should this not be this way?